test/react-native-cli/features/steps/steps.rb

```diff
@@ -209,11 +209,7 @@ def parse_package_json
 end
 
 def get_android_main_application_path(current_fixture)
-  if current_fixture.include? 'expo_ejected'
-    "android/app/src/main/java/com/bugsnag/#{current_fixture}/MainApplication.java"
-  else
-    "android/app/src/main/java/com/#{current_fixture}/MainApplication.java"
-  end
+  "android/app/src/main/java/com/#{current_fixture}/MainApplication.java"
```

Comment:
This is no longer required since the `rn0_63_expo_ejected` fixture has the same format as the rest of the fixtures.